creativecloud/blocks/change-bg/change-bg.js

```diff
@@ -0,0 +1,168 @@
+      groups: [
+        {
+          name: 'Remove Background',
+          iconUrl: `${assetsRoot1}/remove-background-icon.svg`,
+        },
+        {
+          name: 'Change Photo',
+          iconUrl: `${assetsRoot1}/change-photo-icon.svg`,
+          options: [
+            {
+              src: `${assetsRoot}/desktop/photo1.jpg`,
+              swatchSrc: `${assetsRoot}/photo-submenu-1.jpg`,
+            },
+            {
+              src: `${assetsRoot}/desktop/photo2.jpg`,
+              swatchSrc: `${assetsRoot}/photo-submenu-2.jpg`,
+            },
+            {
+              src: `${assetsRoot}/desktop/photo3.jpg`,
+              swatchSrc: `${assetsRoot}/photo-submenu-3.jpg`,
+            },
+          ],
+        },
+        {
+          name: 'Change Color',
+          iconUrl: `${assetsRoot1}/change-color-icon.svg`,
+          options: [
+            { src: '#31A8FF' },
+            { src: '#7F66E6' },
+            { src: '#31F7FF' },
+          ],
+        },
+        {
+          name: 'Change Pattern',
+          iconUrl: `${assetsRoot1}/change-pattern-icon.svg`,
+          options: [
+            {
+              src: `${assetsRoot}/desktop/pattern1.jpg`,
+              swatchSrc: `${assetsRoot}/pattern-submenu-1.jpg`,
+            },
+            {
+              src: `${assetsRoot}/desktop/pattern2.jpg`,
+              swatchSrc: `${assetsRoot}/pattern-submenu-2.jpg`,
+            },
+            {
+              src: `${assetsRoot}/desktop/pattern3.jpg`,
+              swatchSrc: `${assetsRoot}/pattern-submenu-3.jpg`,
+            },
+          ],
+        },
+      ],
+    },
+    tablet: {
+      marqueeTitleImgSrc: `${assetsRoot1}/tablet/everyonecanphotoshop.webp`,
+      talentSrc: `${assetsRoot1}/tablet/yogalady.webp`,
+      defaultBgSrc: `${assetsRoot1}/tablet/defaultBg.webp`,
+      tryitSrc: `${assetsRoot}/tryit.svg`,
+      tryitText: 'Try it',
+      groups: [
+        {
+          name: 'Remove Background',
+          iconUrl: `${assetsRoot1}/remove-background-icon.svg`,
+        },
+        {
+          name: 'Change Photo',
+          iconUrl: `${assetsRoot1}/change-photo-icon.svg`,
+          options: [
+            {
+              src: `${assetsRoot}/tablet/photo1.jpeg`,
+              swatchSrc: `${assetsRoot}/photo-submenu-1.jpeg`,
+            },
+          ],
+        },
+        {
+          name: 'Change Color',
+          iconUrl: `${assetsRoot1}/change-color-icon.svg`,
+          options: [
+            { src: '#31A8FF' },
+          ],
+        },
+        {
+          name: 'Change Pattern',
+          iconUrl: `${assetsRoot1}/change-pattern-icon.svg`,
+          options: [
+            {
+              src: `${assetsRoot}/tablet/pattern1.jpeg`,
+              swatchSrc: `${assetsRoot}/pattern-submenu-1.jpeg`,
+            },
+          ],
+        },
+      ],
+    },
+    mobile: {
+      marqueeTitleImgSrc: `${assetsRoot1}/mobile/everyonecanphotoshop.webp`,
```

Comment:
We can replace these with link to published png images to be fetched in webp format from franklin to improve the performance and reduce load time. The assets hence can be removed from the repo.